frontend/routes/Home.tsx

```diff
@@ -5,7 +5,7 @@ export default function Home() {
   const { isPending, error, data } = useQuery<DWDType>({
     queryKey: ['wetter'],
     queryFn: async () => {
-      const data = await fetch('/wetter').then((res) => res.json())
+      const data = await fetch(`${window.appConfig.basepath}/wetter`).then((res) => res.json())
```

Comment:
I think it would make sense to wrap this into something like `useApiQuery` to avoid having this in every `useQuery` hook.